js/components/Chunk.js

```
@@ -0,0 +1,522 @@
+	 */
+	addToDOM(parent) {
+		if (this.element && parent && !this.element.parentNode) {
+			parent.appendChild(this.element);
+		}
+	}
+
+	/**
+	 * Remove chunk from DOM
+	 */
+	removeFromDOM() {
+		if (this.element && this.element.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)